make/test.mk

```diff
@@ -0,0 +1,3 @@
+.PHONY: test
+test:
+	cd hack/validate-manifests && go run .
```

Comment:
Can we name the file `validate.mk` and target `validate-manifests` to match the tool name?